dist/select/select.css

```diff
@@ -58,16 +58,16 @@ span.select {
   display: none;
 }
 .select select[disabled] {
```

Comment:
The issue of the background color difference in Chrome was because the user agent has an explicit opacity of `0.7` set on the disabled state. I'm not sure why anyone on the chrome dev team thought this was a good idea, but...

<img width="336" alt="image" src="https://user-images.githubusercontent.com/1675667/178021417-d8cd83b0-fc2d-41e3-85a8-ee323b35ba07.png">

Anyway... we should probably add an explicit override for all disabled selects:

`````css
.select select[disabled] {
    opacity: 1;
}
`````
Everything else looks good to me.